feat(changelog): cockpit-added-beta-traces 2023-11-07 (#2378)
scaleway/docs-content: changelog/november2023/2023-11-07-cockpit-added-beta-traces.mdx

---

> title: Beta Traces

Reviewer: ```suggestion
title: Traces are now available in beta
```

status: added
author:

>     fullname: '#observability'

Reviewer 2: ```suggestion
    fullname: 'Join the #observability channel on Slack.'
```

>     url: 'https://scaleway-community.slack.com/archives/C03C81QMC80https://slack.scaleway.com'

Reviewer 2: ```suggestion
    url: 'https://slack.scaleway.com'
```

date: 2023-11-07
category: observability
product: cockpit
---

> Your system's secrets revealed at your fingertips...! Along with metrics and logs, You can now push and pull traces from your Cockpit ! Gain real-time insights and trace the heartbeat of your system with our traces feature based on Grafana Tempo.

Reviewer 2: ```suggestion
Along with metrics and logs, you can now push and pull traces from your Cockpit. Gain real-time insights and trace the heartbeat of your system with our new traces feature, based on Grafana Tempo.
```

> You can find all our documentation to start pushing traces here : https://www.scaleway.com/en/docs/observability/cockpit/how-to/activate-push-traces/

Reviewer: ```suggestion
Read [our documentation](/observability/cockpit/how-to/activate-push-traces/) to find out how to start using our traces feature.
```
